config/sass.js

```diff
@@ -2,9 +2,20 @@ module.exports = {
   options: {
     sourceMap: true,
   },
-  dist: {
+  dev: {
+    options: {
+      outputStyle: 'expanded'
+    },
     files: {
       'public/dist/main.css': 'public/scss/main.scss',
     },
   },
+  min: {
+    options: {
+      outputStyle: 'compressed'
+    },
+    files: {
+      'public/dist/main.min.css': 'public/scss/main.scss',
```

Comment:
Do we need something to toggle prod and link to this? https://github.com/mozilla/payments-ui/blob/master/public/index.html#L7 Or is that for a later commit?